src/crosswalk/plots.py

```diff
@@ -54,14 +54,22 @@ def dose_response_curve(dose_variable, obs_method,
     "All covariates in cwmodel should be specified in " \
     "dose_variable or continuous_variables or binary_variables!"
 
+    # Modif Ariane: Join alt_dorms here to be able to use numpy ravel when creating the data frame
+    alt_dorms = cwdata.alt_dorms
+    formatted_alt_dorms = [cwdata.dorm_separator.join(x) for x in cwdata.alt_dorms]
+    cwdata.alt_dorms = formatted_alt_dorms
```

Comment:
Thanks @ADucellierIHME! Minor suggestion, I think we don't need to overwrite the `cwdata.alt_dorms`, and replace `np.ravel(cwdata.alt_dorms)` to `np.ravel(formatted_alt_dorms)` below.